utils/frame/try-runtime/remote-externalities/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[package]
+name = "remote-externalities"
+version = "3.0.0"
```

Comment:
```suggestion
version = "0.8.0"
```

Reply:
Oh well but we recently updated unstable substrate stuff to 0.9.0. Let's not make Ben unhappy and use 0.9.0